--- memcached-1.5.4/Makefile.am.0000~	2017-12-22 22:59:31.523021440 +0100
+++ memcached-1.5.4/Makefile.am	2017-12-22 23:00:14.314142999 +0100
@@ -110,26 +110,6 @@ test:	memcached-debug sizes testapp
 	else \
 	  prove $(srcdir)/t; \
 	fi
-	@if test `basename $(PROFILER)` = "gcov"; then \
-	  for file in memcached_debug-*.gc??; do \
-	    mv -f $$file `echo $$file | sed 's/memcached_debug-//'`; \
-	  done && \
-	  for file in *.gcda; do \
-	    srcfile=`echo $$file | sed 's/.gcda/.c/'`; \
-	    if test -n "`echo $(memcached_debug_SOURCES) | grep $$srcfile`"; then \
-	      echo `$(PROFILER) $$srcfile` | sed 's/'$$srcfile':.*//'; \
-	    fi \
-	  done \
-	elif test `basename $(PROFILER)` = "tcov"; then \
-	  files=`grep SRCFILE memcached-debug.profile/tcovd | sed 's/SRCFILE://' | sort | uniq` && \
-	  $(PROFILER) -x memcached-debug.profile $$files 2>&1; \
-	  for file in *.tcov; do \
-	    srcfile=`echo $$file | sed 's/.tcov//'`; \
-	    if test -n "`echo $(memcached_debug_SOURCES) | grep $$srcfile`"; then \
-	      echo $$srcfile : `grep 'Percent of the file executed' $$file`; \
-	    fi \
-	  done \
-	else :; fi
 
 docs:
 	(cat Doxyfile ; echo "PROJECT_NUMBER=`cat version.num`") | doxygen -
